packages/validator/test/cli-validator/tests/configuration-manager.test.js

```diff
@@ -400,6 +405,22 @@ describe('Configuration Manager tests', function () {
         }
       );
 
+      it.each(['-q', '--quality-score'])(
+        `should produce correct config with -q/--quality-score option`,
+        async function (option) {
+          const expectedConfig = {
+            produceQualityScore: true,
+          };
+
+          const { context } = await processArgs([option], cliParseOptions);
+          const capturedText = getCapturedText(consoleSpy.mock.calls);
+          // originalError(`Captured text: ${JSON.stringify(capturedText, null, 2)}`);
```

Comment:
Do we need this here?

Reply:
IMO, it never hurts to have that comment there in case we need to really really debug the testcase at some point in the future :)

Reply:
Agreed - the point is to make it easily "uncommented" if we need to check the output we're performing assertions on, should it change, etc.